modules/pages/partials/docker.adoc

```diff
@@ -0,0 +1,8 @@
+This SDK runs on top of the {cpp} core, Couchbase++, which requires a {cpp} 17 compiler, such as GCC 8.0, or more recent versions.
```

Comment:
We can use the {[cbpp](https://github.com/couchbase/docs-site/blob/master/antora-playbook.yml#L161)} variable for `Couchbase++`